docs/introduction-to-smart-contracts.rst

```diff
@@ -71,40 +71,41 @@ so that only you can alter the number.
 Subcurrency Example
 ===================
 
-The following contract will implement the simplest form of a
-cryptocurrency. It is possible to generate coins out of thin air, but
-only the person that created the contract will be able to do that (it is easy
-to implement a different issuance scheme).
-Furthermore, anyone can send coins to each other without a need for
-registering with username and password — all you need is an Ethereum keypair.
-
+The following contract implements the simplest form of a
+cryptocurrency. The contract allows only its creator to create new coins (different issuance scheme are possible).
+Anyone can send coins to each other without a need for
+registering with a username and password, all you need is an Ethereum keypair.
 
 ::
 
     pragma solidity >=0.5.0 <0.7.0;
 
     contract Coin {
-        // The keyword "public" makes those variables
-        // easily readable from outside.
+        // The keyword "public" makes variables
+        // accessible from other contracts
         address public minter;
         mapping (address => uint) public balances;
 
-        // Events allow light clients to react to
-        // changes efficiently.
+        // Events allow clients to react to specific
+        // contract changes you declare
         event Sent(address from, address to, uint amount);
 
-        // This is the constructor whose code is
-        // run only when the contract is created.
+        // Constructor code is only run when the contract
+        // is created
         constructor() public {
             minter = msg.sender;
         }
 
+        // Sends an amount of newly created coins
+        // from the contract creator to an address
```

Comment:
The contract creator does not really own them, so they are not sent from the contract creator.

Reply:
Creates an amount of coins and assigns them to a specified address?

Reply:
(can only be called by the contract creator)